src/display/lv_display.h

```diff
@@ -227,7 +227,16 @@ int32_t lv_display_get_dpi(const lv_display_t * disp);
  *--------------------*/
 
 /**
- * Set the buffers for a display
+ * Set the buffers for a display, similarly to `lv_display_set_draw_buffers`, but accept the raw buffer pointers.
+ * @param disp              pointer to a display
+ * @param buf1              first buffer
+ * @param buf2              second buffer (can be `NULL`)
```

Comment:
`buf_size` and `render__mode` parameters are missing

Reply:
Added in https://github.com/lvgl/lvgl/pull/5281